vcpkg-init/vcpkg-init.ps1

```diff
@@ -99,6 +109,8 @@ function bootstrap-vcpkg {
   $PATH = $ENV:PATH
   $ENV:PATH="$VCPKG_ROOT;$PATH"
 
+  Set-Content -Path $VCPKG_VERSION_MARKER -Value $VCPKG_INIT_VERSION -Force -NoNewline
```

Comment:
```suggestion
  Set-Content -Path $VCPKG_VERSION_MARKER -Value "$VCPKG_INIT_VERSION`n" -Force -NoNewline
```

nit

I prefer newline files since git is happier with them.

Reply:
This file is never going to go into a git repo though